activiti-common-converters/pom.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.activiti.core.common</groupId>
+    <artifactId>activiti-core-common-parent</artifactId>
+    <version>7.1.0-SNAPSHOT</version>
+  </parent>
+  <artifactId>activiti-common-converters</artifactId>
```

Comment:
Minor one:
converters is quite specific, would "util" work better in case we will need to add something else too in the future?

Reply:
You mean to rename it to 
**activiti-common-util**?
Is this a final name?

Reply:
@erdemedeiros any preferences?

Reply:
I have made a refactoring and now we will have:
activiti-common-util
The test for DateformatterProvider will live in Activiti:
https://github.com/Activiti/Activiti/blob/develop/activiti-spring-process-extensions/src/test/java/org/activiti/spring/process/variable/DateFormatterProviderTest.java
In this way it's better scoped with variables...

Reply:
Generally speaking, I tend to avoid `util` in module or java class names, as this often end up in places where people put all kind of weird things that they don't know where to put. However, here, I'm struggling to find a better name, so let's go with `util`.